core-bundle/src/Resources/config/services.yml

```diff
@@ -268,6 +268,17 @@ services:
         tags:
             - { name: data_collector, template: '@ContaoCore/Collector/contao.html.twig', id: contao }
 
+    Contao\CoreBundle\Doctrine\Backup\DumperInterface:
+        class: Contao\CoreBundle\Doctrine\Backup\Dumper
+
+    Contao\CoreBundle\Doctrine\Backup\BackupManager:
+        arguments:
+            - '@database_connection'
+            - '@Contao\CoreBundle\Doctrine\Backup\DumperInterface'
+            - '%kernel.project_dir%/var/backups' # Set by extension
+            - [tl_crawl_queue, tl_log, tl_search, tl_search_index, tl_search_term] # Set by extension
+            - 5 # Set by extension
```

Comment:
When are these values used if the extension sets them anyway?

Reply:
Because @aschempp wanted them to be here :)

Reply:
Does not make any sense to me. 🤷‍♂️

Reply:
Imho it should just be like in other places so if we don't do it elsewhere, we shouldn't do it here either. So that's a CS fix as well imho.

Reply:
I'd vote for removing. Writing `Set by extension` and - at the same line - still setting data is kind of incoherent communication.

Reply:
Especially since PHPStorm already shows the argument type:

<img width="480" alt="" src="https://user-images.githubusercontent.com/1192057/143192073-0f9d60c0-04e6-491d-8761-3323809709c1.png">